hack/provisioning/linux/containerd.sh

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+
+#   Copyright The containerd Authors.
+
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+
+#       http://www.apache.org/licenses/LICENSE-2.0
+
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+set -o errexit -o errtrace -o functrace -o nounset -o pipefail
+root="$(cd "$(dirname "${BASH_SOURCE[0]:-$PWD}")" 2>/dev/null 1>&2 && pwd)"
+readonly root
+# shellcheck source=/dev/null
+. "$root/../../scripts/lib.sh"
+
+# provision::containerd::uninstall will ensure deb containerd are purged
+provision::containerd::uninstall(){
+  [ "$(id -u)" == 0 ] || {
+    log::error "You need to be root"
+    return 1
+  }
+
+  # Purge deb package
+  apt-get -q purge containerd 2>/dev/null || true
```

Comment:
```suggestion
  apt-get -q purge containerd* 2>/dev/null || true
```
Pkg can be containerd.io

Reply:
Ok.